ui/desktop/electron-app/src/ipc/handlers.js

```diff
@@ -44,9 +44,10 @@ handle('resetClusterUrl', async () => runtimeSettings.resetClusterUrl());
  * testing workflows).  Insecure URLs are allowed in dev mode.
  */
 handle('openExternal', async (href) => {
+  const localhostRegex = /^http:\/\/(localhost|127\.0\.0\.1):\d{1,5}(?:\/|$)/i;
```

Comment:
`http://localhost:9200` is what we want to confirm as to not allow a URL like `http://localhost.somedomain.com/` or `http://localhostdomain.com/`. However, the URL it is checking against does have a subpath like this, `http://127.0.0.1:50911/authorize?client_id=...`. So I added the last part `(?:\/|$)` because without it the check was failing. Is there a better way to achieve that?

Reply:
Ahh yup that's fine then